buildScript/webpack.config.js

```diff
@@ -106,54 +107,88 @@ export default function makeWebpackConfig(config) {
 
     if (PROD) {
         plugins.push(
-            new webpack.optimize.OccurrenceOrderPlugin(),
-            new webpack.optimize.DedupePlugin(),
-            new webpack.optimize.UglifyJsPlugin({
-                compress : {
-                    warnings  : false,
-                    unused    : true,
-                    dead_code : true
+            // if using the latest uglifyjs-webpack-plugin, based on UglifyJSv3, understanding es6 modules
+            // this uglifier also minimizes the code, no LoaderOptionsPlugin is needed
+            new UglifyJsPlugin({
```

Comment:
i agree that using uglify 3 is the right decision when  it will be standard in webpack 4